backend/DailyCheckInAPIs/insert_daily.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+
+namespace DailyCheckInAPIs
+{
+    public static class insert_daily
+    {
+        // The Azure Cosmos DB endpoint for running this sample.
+        private static readonly string EndpointUri = Environment.GetEnvironmentVariable("COSMOS_ENDPOINT");
+
+        // The primary key for the Azure Cosmos account.
+        private static readonly string PrimaryKey = Environment.GetEnvironmentVariable("PKEY");
+
+        // The Cosmos client instance
+        private static CosmosClient cosmosClient;
+
+        // The database we will create
+        private static Database database;
+
+        // The container we will create.
+        private static Container container;
+
+        // The name of the database and container we will create
+        private static string databaseName = "coronafear";
+        private static string containerName = "dailysurvey";
+
+        // Azure function triggered when user submits his/her data via DCI Mobile App
+        [FunctionName("insert_daily")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "get", "post", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            log.LogInformation("insert_daily function processed a request.");
+
+            DailySurveyData dailySurveyData = await insert_daily.PopulateDailySurveyDataFromHttpRequest(req);
+            dailySurveyData.EntryDate = DateTime.Today.Date.ToString();
```

Comment:
Can we get rid of entry date population here if that's happening inside PopulateDailySurveyDataFromHttpRequest method?